portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+img{
+    width: 25%;
+    height: auto;
+}
```

Comment:
This will work for this purpose, but you may run into some issues in future changes.  This will give every img tag a width corresponding to 25% of it's parent element, which probably isn't what you'd want in general.  I'd recommend changing 2 things:

1) Use CSS class selectors (See https://www.w3schools.com/cssref/css_selectors.asp) over element selectors.  These are much more flexible, and allow you to reuse styles across different types of elements and merge different styles
2) Use a fixed length unit (See https://www.w3schools.com/cssref/css_units.asp).  Percentages are based on the size of the parent, which can frequently change as you add layers of containers.  For images I usually go with fixed px (pixels) or em (width of an "m" in the current font).  However, vw and vh are also very useful when you want to support different sized screen.  They represent 1% of the windows width/height, so "width: 25vw" should give you the same behavior you have now

Reply:
ok thank you.